addon/components/session/collapsed-objectives.hbs

```diff
@@ -1,13 +1,13 @@
 <section
   class="session-collapsed-objectives"
   data-test-session-collapsed-objectives
-  {{did-insert (perform this.load) @session.sessionObjectives}}
+  {{did-insert (perform this.load)}}
```

Comment:
see suggested changes to `course/collapsed-objectives.hbs` above 

`{{did-insert (perform this.load) @session.sessionObjectives}}`